site/scripts/sync-algolia-index.ts

```diff
@@ -19,6 +19,12 @@ type DocsFrontMatter = {
 
 type BlockProtocolFileTypes = "spec" | "docs";
```

Comment:
Do we still need this?

Reply:
It is referred to in 3 places below

Reply:
I’m wondering if there still exists conceptual necessity in this type, given that `spec` is a sub-path of `/docs`

Reply:
Ah, right – the designs for results still distinguish between 'spec' results and 'docs' results (in breadcrumbs), so I think it's useful to keep them distinct, yes